sites/all/modules/custom/ecas/ecas_import_users/ecas_import_users.info

```diff
@@ -2,7 +2,8 @@ name = ECAS import users
 description = Create users accounts from ECAS accounts (in LDAP).
 package = "ECAS"
 core = 7.x
-php = 5.0
```

Comment:
I've only checked the code related to the bug fix of the JIRA MULTISITE-5080.
I've tested it, it's ok, I approve the pull request.

The coding standard fixes should be done in another jira task.